setup.py

```diff
@@ -53,7 +53,19 @@
     setupext.BackendMacOSX(),
     ]
 
-
+if sys.version_info < (3, 6):
```

Comment:
We will never get here because we will fail on L14 due to `if sys.version_info < py_min_version:` failing for any Python below 3.7.